default-cluster-templates/baseline-k3s.json

```diff
@@ -0,0 +1,105 @@
+  "controlplaneprovidertype": "k3s",
+  "infraprovidertype": "intel",
+  "clusterconfiguration": {
+    "kind": "KThreesControlPlaneTemplate",
+    "apiVersion": "controlplane.cluster.x-k8s.io/v1beta1",
+    "metadata": {
+      "labels": {
+        "cpumanager": "true"
+      }
+    },
+    "spec": {
+      "template": {
+        "spec": {
+          "files": [
+            {
+              "path": "/var/lib/rancher/k3s/agent/etc/containerd/config.toml.tmpl",
+              "content": "version = 2\n\n[plugins.\\\"io.containerd.internal.v1.opt\\\"]\n  path = \\\"/var/lib/rancher/k3s/agent/containerd\\\"\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\"]\n  stream_server_address = \\\"127.0.0.1\\\"\n  stream_server_port = \\\"10010\\\"\n  enable_selinux = false\n  enable_unprivileged_ports = true\n  enable_unprivileged_icmp = true\n  sandbox_image = \\\"index.docker.io/rancher/mirrored-pause:3.6\\\"\n  disable_apparmor = true\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".containerd]\n  snapshotter = \\\"overlayfs\\\"\n  disable_snapshot_annotations = true\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".containerd.runtimes.runc]\n  runtime_type = \\\"io.containerd.runc.v2\\\"\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".containerd.runtimes.runc.options]\n  SystemdCgroup = true\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".registry]\n  config_path = \\\"/var/lib/rancher/k3s/agent/etc/containerd/certs.d\\\"\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".containerd.runtimes.kata-qemu]\n  runtime_type = \\\"io.containerd.kata-qemu.v2\\\"\n  runtime_path = \\\"/opt/kata/bin/containerd-shim-kata-v2\\\"\n  privileged_without_host_devices = true\n  pod_annotations = [\\\"io.katacontainers.*\\\"]\n\n[plugins.\\\"io.containerd.grpc.v1.cri\\\".containerd.runtimes.kata-qemu.options]\n  ConfigPath = \\\"/opt/kata/share/defaults/kata-containers/configuration-qemu.toml\\\""
+            }
+          ],
+          "agentConfig": {
+            "kubelet": {
+              "extraArgs": [
+                "--topology-manager-policy=best-effort",
+                "--cpu-manager-policy=static",
+                "--reserved-cpus=1",
+                "--max-pods=250",
+                "--tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
+              ]
+            }
+          },
+          "preK3sCommands": [
+            "mkdir -p /etc/systemd/system/k3s-server.service.d",
+            "echo '[Service]\nEnvironmentFile=/etc/environment' > /etc/systemd/system/k3s-server.service.d/override.conf",
+            "make -p /var/lib/kubelet/static-pods",
+            "sed -i '/kubelet-arg:/a\\- pod-manifest-path=/var/lib/kubelet/static-pods' /etc/rancher/k3s/config.yaml"
+          ],
+          "postK3sCommands": [
+            "kubectl apply -f https://raw.githubusercontent.com/projectcalico/calico/v3.26.1/manifests/calico.yaml"
```

Comment:
i see calico v3.26.X does not directly support kubernetes 1.30.X The support ends at 1.28
https://archive-os-3-26.netlify.app/calico/3.26/getting-started/kubernetes/requirements#kubernetes-requirements

I believe it would be better to bump to v3.29
https://docs.tigera.io/calico/3.29/getting-started/kubernetes/requirements#kubernetes-requirements
![image](https://github.com/user-attachments/assets/3e3abcbf-56bf-40ac-b107-ee21d3ab9734)

Reply:
good catch! yes, agree

Reply:
changes in [3a891b2](https://github.com/open-edge-platform/cluster-manager/pull/62/commits/3a891b20e357c74cdc9fddc673040f0fac9ee582)